README.md

```diff
@@ -118,6 +118,7 @@ Note: All O3DE projects generate a GameLauncher and a ServerLauncher. NetSoakTes
 | soak_port | The port that this soak test will bind to for game traffic | 33450 |
 | soak_protocol | Soak test protocol (TCP or UDP) | udp | 
 | soak_mode | The operating mode for the soak test, options are loopback, client or host. `Loopback` has two connection within the application feed traffic to each other in a loop. `Client` expects to connect to a server hosted at soak_serveraddr. `Host` hosts a server for clients to connect to | Loopback | 
+| DumpSoakStats | Dump snapshot of soak test networking stats to console | N/A|
```

Comment:
NP: Missing a trailing space after N/A. That might not produce any difference in the README

Reply:
Happy to fix that in next change, if ok as am adding option.